azurerm/internal/services/apimanagement/resource_arm_api_management.go

```diff
@@ -100,6 +103,15 @@ func resourceArmApiManagementService() *schema.Resource {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
+						"identity_ids": {
+							Type:     schema.TypeList,
```

Comment:
Could we change this type to `TypeSet`? The list of user assigned identity ids is stored as a map in the REST API request (therefore the SDK does the same), which indicates this should be an un-ordered list.
If you switched the type to TypeSet, the expand and flatten code may need changes accordingly.

Reply:
That sounds reasonable, I've gone and made that change in 4d1f05b.